finalProject/tenSpeckledFrogs.js

```diff
@@ -0,0 +1,22 @@
+
+
+
+
+
+
+
+
+
+for (var i = 10; i > 0; i--) {
```

Comment:
developers tend to use i++ when possible. Here, that makes you need to do math inside your interpolation so I understand using i--, but I'd say using i++ is still more common.
The next step with this challenge is to set this up in a function that takes an argument so it can be run for any number of frogs.